frontend/rolecall/src/app/user/user-editor.component.html

```diff
@@ -48,13 +57,27 @@ <h1 class="right-card-title">
               [initValue]="(currentSelectedUser | emptyStringIfUndefined)['first_name'] | emptyStringIfUndefined">
           </app-text-input>
         </div>
+        <div class="name-input">
+          <app-text-input
+              valueName="Middle Name"
+              (valueChange)="onInputChange($event)"
+              [initValue]="(currentSelectedUser | emptyStringIfUndefined)['middle_name'] | emptyStringIfUndefined">
```

Comment:
This will produce invalid code if `currentSelectedUser` is actually undefined, because it will try to take a substring from an empty string.

The proper thing to do would be to use null-safe property access:
```
[initValue]="currentSelectedUser?.middle_name || ''"
```

Reply:
Fixed.